pkg/cluster/install.go

```diff
@@ -62,6 +63,12 @@ func (m *manager) adminUpdate() []steps.Step {
 		)
 	}
 
+	if isEtcdRecovery {
```

Comment:
do we need this as a part of `adminUpdate`/PUCM if we exclusively enact it only as a maintenance task? IMO this might be better served as a separate Geneva Action endpoint.

Reply:
No I don't think so. I agree that a separate endpoint is probably best since it's so specific, and not included in everything.

Reply:
I am +1 on just adding the recovery code in this PR, and then using a separate PR for adding a geneva action endpoint to execute it.